src/controllers/promo_codes/post.docs.ts

```diff
@@ -0,0 +1,31 @@
+/**
```

Comment:
@bram-rongen Separate file for swagger docs, suggested `*.docs.ts` as a filename but open to alternatives:) 

`*.spec.ts` is usually meant for tests right? The name kinda aligns with what this stuff is meant for, but hesitant to use it.

Reply:
docs seems better indeed!